### DISCUSSION TITLE
Expose `cacheExtent` for `ChatList`

### DISCUSSION THREAD

**Discussion by Brandon-Wolff** [flyerhq/flutter_chat_ui#537] [Q&A] [answered]
When there are several image messages in a row, scrolling back and forth causes them to reload even when the message is barely off screen. With the default image message builder, it causes a lot of jumping around as images load in. `CachedNetworkImage` will still display a progress indicator for a moment when the message enters the screen, but adding a sufficient `cacheExtent` value to the `CustomScrollView` fixes that.

**Comment by demchenkoalex** [comment]
Hey, current version is not optimised at all, and instead of adding properties I am working on a full re-design with the `v2` version. Not jumping messages and cache will be in the core of the solution, making chat screen performant and smooth.

**Comment by demchenkoalex** [comment, marked as answer]
cache built-in in v2 https://github.com/flyerhq/flutter_chat_ui/issues/646
